runtime/src/main/java/com/flipkart/flux/guice/module/ConfigModule.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2012-2016, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.flux.guice.module;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Iterator;
+
+import com.flipkart.polyguice.config.ApacheCommonsConfigProvider;
+import com.flipkart.polyguice.config.YamlConfiguration;
+import com.google.inject.AbstractModule;
+import com.google.inject.binder.LinkedBindingBuilder;
+import com.google.inject.name.Names;
+
+/**
+ * <code>ConfigModule</code> is a Guice {@link AbstractModule} implementation used for wiring flux configuration.
+ * @author kartik.bommepally
+ */
+public class ConfigModule extends AbstractModule {
+
+    private static final String CONFIGURATION_YML = "packaged/configuration.yml";
```

Comment:
Keep this in RutimeConstants file, in this way we can have all variables in one place.